packages/types-terra/src/project.ts

```diff
@@ -0,0 +1,144 @@
+// Copyright 2020-2021 OnFinality Limited authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import {BlockInfo, EventsByType, LCDClient} from '@terra-money/terra.js';
```

Comment:
If we can limit to just type imports that would be ideal. Same applies to other files in terra-types

```suggestion
import types{BlockInfo, EventsByType, LCDClient} from '@terra-money/terra.js';
```